pages/common/fd.md

```diff
@@ -23,3 +23,7 @@
 - Include ignored and hidden files in the search:
 
 `fd --hidden --no-ignore {{pattern}}`
+
+- Execute command on search results:
```

Comment:
Perhaps this will read better?
```suggestion
- Execute a command on each search result returned:
```

I assume this keeps the same meaning?

Reply:
Agreed! Updated + rebased.